cloudify_cli/cli/helptexts.py

```diff
@@ -108,6 +108,9 @@
     "Terminate the execution abruptly, rather than request an orderly "
     "termination"
 )
+KILL_EXECUTION = (
+    "Terminate the execution abruptly, and also stop currently running tasks"
```

Comment:
Maybe add a word about the fact that we're actually killing user processes here, and not just Cloudify's internal executions?